app/components/PhotoUploadStatus/index.js

```diff
@@ -120,13 +120,3 @@ const ConnectedUploadStatusCard = connect(
 )(UploadStatusCard);
 
 export default ConnectedUploadStatusCard;
-
-export const addUploadStatusCard = <T>(Component: ComponentType<T>) => (
```

Comment:
Why remove this?

Reply:
@LudvigHz This component was created 2 years ago, and is never used anywhere. Thus, there is no point having it in the codebase.

Reply:
Ok, nice :ok_hand: